js/standalone.js

```diff
@@ -20,44 +20,55 @@ var inputs = {
 	suffix: document.getElementById('option-suffix')
 };
 
-var buttons = {
+const buttons = {
 	copy: document.getElementById('btn-copy')
 };
 
 function initUi() {
 	availableAlgos.forEach(function (algo) {
-		var el = document.createElement('option');
+		const el = document.createElement('option');
 		el.text = algo;
 		el.value = algo;
-		inputs.hashAlgorithm.appendChild(el);
+		if (inputs.hashAlgorithm) {
+			inputs.hashAlgorithm.appendChild(el);
+		}
 	});
 
-	form.addEventListener('submit', function (event) {
-		event.preventDefault();
-		generatePassword();
-	});
+	if (form) {
```

Comment:
-1 to those checks. I prefer to fail badly than to silently ignore these.

Reply:
Should I add a function `fail_badly` ? It would open a big blinking pop-up or something like this.